frontend/rpm/handle_sources.go

```diff
@@ -77,6 +77,13 @@ func buildScript(spec *dalec.Spec) string {
 	fmt.Fprintln(b, "set -e")
 
 	if spec.HasGomods() {
+		// Older go versions did not have support for the `GOMODCACHE` var
+		// This is a hack to try and make the build work by linking the go modules
+		// we've already fetched into to module dir under $GOPATH
```

Comment:
Can you add a comment mentioning that `${GOPATH}/pkg/mod` is the default module dir?